tests/should-pass/iati-activities/iati-activity/result/indicator/document-link/title/element-present-with-namespace-attribute.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+
+<iati-activities version="xx">
+  <iati-activity xmlns:example-namespace="http://example.org/example-namespace/ns#"><!-- Include namespace declaration. -->
+    <iati-identifier></iati-identifier>
+    <reporting-org type="xx" ref="xx">
+      <narrative>Organisation name</narrative>
+    </reporting-org>
+    <title>
+      <narrative>Xxxxxxx</narrative>
+    </title>
+    <description>
+      <narrative>Xxxxxxx</narrative>
+    </description>
+    <participating-org role="xx"></participating-org>
+    <activity-status code="xx"/>
+    <activity-date type="xx" iso-date="2013-11-27"/>
+    <activity-date type="xx" iso-date="2013-11-27">
+      <narrative>Some stuff ere</narrative>
+    </activity-date>
+    <result type="xx">
+      <title>
+        <narrative>Sample text</narrative>
+      </title>
+      <indicator measure="xx" ascending="0">
+        <title>
+          <narrative>Sample text</narrative>
+        </title>
+        <document-link url="xx" format="xx">
+          <title example-namespace:namespace-attribute="xx"><!-- PASS: Optional element added with required child element and namespace attribute. -->
+            <narrative>Sample text</narrative>
+          </title>
+          <category code="xx" />
```

Comment:
whitespace, might want to do a check throughout as it always seems to be with the code attribute.

Reply:
This was resolved in caf1ce7.